cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml

```diff
@@ -242,6 +243,14 @@ Parameters:
         AllowedPattern: '^(.*|[0-9]+)$'
         ConstraintDescription: >-
           When specified, Auto-scaling Group Warm Pool Minimum Size should be a whole number
+      AsgS3BucketName:
```

Comment:
i'm confused. are we hosting a public S3 bucket with the zip file for all customers to pull from? if so, why do i need a copy in tf github?

Reply:
No, we are not using a public s3 bucket. The user workflow for CFT is that, they get the template and the zip file when they download the release from github. After that, the user uploads the lambda zip file into an s3 bucket in their org etc and then, when running through the cloudformation parameters, they enter the s3 bucket details for cloudformation lambda resource to use.